frontend/Realm.js

```diff
@@ -178,6 +178,15 @@
                 app.ports.changePage.subscribe(changePage);
             }
 
+            if (app.ports) {
+                for (let [portName, port]  of Object.entries(app.ports)) {
```

Comment:
You are using let which is only supported in modern browsers. As of now we do not have a convert modern to old dependency, so let's using old javascript as much as can.

To check for browser support use https://caniuse.com/#search=let. Let in loop is problematic in IE for example. Object.entries not supported at all in IE.

Reply:
We should also store app on say window.realm_app.